### ISSUE TITLE
Smart scroll

### ISSUE THREAD

**Issue by anka-213** [anka-213/webcomic_reader#5]
[This page](http://mangapark.me/manga/boku-girl-sugito-akira/s5/c57/1) has a pretty clever script for scrolling and navigating when the comic doesn't fit on the screen. Perhaps steal the idea?

**Comment by JonasNo** [comment]
Is the script gone from that page?

**Comment by anka-213** [comment]
@JonasNo Nope, you just have to disable WCR first. Otherwise it's overridden.
